tarrif/src/main/java/pl/edu/agh/tarrif/tarrif/TarrifController.java

```diff
@@ -0,0 +1,34 @@
+package pl.edu.agh.tarrif.tarrif;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.*;
+import pl.edu.agh.tarrif.auth.CurrentUser;
+import pl.edu.agh.tarrif.auth.User;
+import pl.edu.agh.tarrif.tarrif.dto.PricingRecord;
+import pl.edu.agh.tarrif.tarrif.service.TarrifService;
+import javax.validation.Valid;
+import java.util.ArrayList;
+
+@CrossOrigin(origins = "*")
+@RestController
+@RequestMapping("/api")
+public class TarrifController {
+    private final TarrifService tarrifService;
+
+    public TarrifController(final TarrifService tarrifService) {
+        this.tarrifService = tarrifService;
+    }
+
+    @GetMapping("/pricing")
+    @ResponseStatus(HttpStatus.OK)
+    public ArrayList<PricingRecord> getPricing(@CurrentUser final User user){
```

Comment:
If you are not using user in a function in controller, you don't have to pass it in an argument. Spring inserts the user if you include annotation CurrentUser. I suggest deleting this if possible.

Reply:
Removed the user argument, as it was not used